siliconcompiler/core.py

```diff
@@ -3775,14 +3775,43 @@ def read(fd):
                     import pty # Note: this import throws exception on Windows
                     retcode = pty.spawn(cmdlist, read)
             else:
-                with open(logfile, 'w') as log_writer, open(logfile, 'r') as log_reader:
+                stdoutFile = ''
+                if self.get('eda', tool, 'stdout', step, index, 'destination') == 'log':
+                    stdoutFile = step + "." + self.get('eda', tool, 'stdout', step, index, 'suffix')
```

Comment:
These lines are getting a little long, would recommend storing `suffix` in a temp variable to keep them shorter, e.g.:

```suggestion
                    suffix = self.get('eda', tool, 'stdout', step, index, 'suffix')
                    stdout_file = step + "." + suffix
```